packages/odyssey-react-mui/package.json

```diff
@@ -7,11 +7,11 @@
   "sideEffects": false,
   "main": "dist/index.js",
   "module": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "types": "dist/src/index.d.ts",
```

Comment:
src folder in dist? That is not common, right? Can we avoid that?

Reply:
Yes. Only the TS files because `isolateModules: true`.

Reply:
That's how it was the whole time. I don't think it's easy or necessary to fix.